### PR TITLE
Add cancel button to password change menu.

### PR DESCRIPTION
**Pull request by red-001**
Before:
old_password_change
After:
new_password_change

### CODE REVIEW

**Comment by SmallJoker** [comment]
Should be consistent with the mainmenu buttons.

File: src/guiPasswordChange.cpp [hunk 1]
```diff
@@ -71,6 +72,11 @@ void GUIPasswordChange::removeChildren()
 		if(e != NULL)
 			e->remove();
 	}
+	{
+		gui::IGUIElement *e = getElementFromId(ID_cancel);
+		if(e != NULL)
```
Comment (SmallJoker) [inline src/guiPasswordChange.cpp]: Space after `if`

File: src/guiPasswordChange.cpp [hunk 2]
```diff
@@ -150,12 +156,19 @@ void GUIPasswordChange::regenerateGui(v2u32 screensize)
 
 	ypos += 50;
 	{
-		core::rect<s32> rect(0, 0, 140, 30);
-		rect = rect + v2s32(size.X/2-140/2, ypos);
+		core::rect<s32> rect(0, 0, 100, 30);
+		rect = rect + v2s32(size.X/2/2+185, ypos);
```
Comment (SmallJoker) [inline src/guiPasswordChange.cpp]: `size.X / 4 + 185, ypos` (style wise)

**Comment by nerzhul** [comment]
merging in ~3h

**Comment by Zeno-** [comment]
Seems fine. :+1:

**Pull request merged**